### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by arneschreuder** [COS-301/graduates#16] [Ideas]
Welcome to the Ideas Discussions.

**Comment by DamianJordaan** [comment]
GitHub Wiki Discussion?
Standards for documentation of Features and teams needs improvement, there should not be a page for every single feature or they should all be sub heading for all the features, there is also a lot of duplicate/redundant information on the wiki.

@arneschreuder is the above logic correct?

**Reply by arneschreuder** [reply]
That is absolutely correct!
